hackathon_site/dashboard/frontend/src/slices/order/orderSlice.ts

```diff
@@ -0,0 +1,54 @@
+import {
+    createAsyncThunk,
+    createEntityAdapter,
+    createSelector,
+    createSlice,
+    PayloadAction,
+} from "@reduxjs/toolkit";
+import { Order } from "api/types";
+import { get } from "api/api";
+import { RootState } from "../store";
+import { cartReducerName } from "../hardware/cartSlice";
+
+interface OrderExtraState {
+    isLoading: boolean;
+    error: string | null;
```

Comment:
if you have a state, you should make a selector for it
So could you make a selector for error as well